include/cooking/cookingJavascript/searchbar.php

```diff
@@ -0,0 +1,20 @@
+<script>
+function searchBar() {
+    // Declare variables
+    var input, filter, ul, li, a, i;
+    input = document.getElementById('myInput');
+    filter = input.value.toUpperCase();
+    ul = document.getElementById("myUL");
```

Comment:
The way you're grabbing elements looks good here. If you want a challenge, try making this into a more re-usable search framework. Like, make it so this function takes a few parameters (like, the id of the input field and the id of the list) and handles search for whatever happens to be contained inside those. Consider what types of other parameters this function might take to customize the search. Consider if you could make a PHP function that takes some arbitrary data and prints it in a way that makes it compatible with this search.